docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -60,6 +61,26 @@ function_call ::= function_name '(' [ arguments ... ] ')'
 | MIN | Returns the minimum value of column values |
 | MAX | Returns the maximum value of column values |
 
+
+## Partition_hash function
+YugabyteDB uses `partition_hash` function that takes as arguments the values of the `partition keys` and returns an `uint16` 
+hash used to distribute rows internally. The full hash partition value spans from `0 - 65535`. 
+We can use this function to query a subset of the data, get approximate count of rows in table, or do a parallel full table scans.
+
+### Querying a subset of the data
+Assuming we have a table:
+```sql
+create table t (h1 int, h2 int, r1 int, r2 int, v int, 
+                         primary key ((h1, h2), r1, r2));
+```
+We can use this function to query a subset of the data (in this case, 1/128 of the data):
+```sql
+select count(*) from t where partition_hash(h1, h2) >= 0 and
+                                      partition_hash(h1, h2) < 512;
```

Comment:
You don't explain how you got 512 in the description (as I had mentioned in the earlier iteration of feedback).